HousingRepairsSchedulingApi.Tests/UseCasesTests/BookAppointmentUseCaseTests.cs

```diff
@@ -80,6 +80,23 @@ public BookAppointmentUseCaseTests()
             await act.Should().ThrowExactlyAsync<T>();
         }
 
+        [Fact]
+#pragma warning disable xUnit1026
+#pragma warning disable CA1707
+        public async void GivenAnInvalidOrderComments_WhenExecute_ThenExceptionIsThrown()
```

Comment:
What is this testing? i.e. what makes these order comments invalid?

I believe it's because `orderComments` is more than 255 characters.
Updating the name to reflect the input would help understand what this unit test is testing.
```suggestion
        public async void GivenOrderCommentsLongerThan255Characters_WhenExecute_ThenExceptionIsThrown()
```